### ISSUE TITLE
Bug with set css style on active control

### ISSUE THREAD

**Issue by zendre4** [pradosoft/prado#800]
Hello,

I found a bug with active controls. When set style on the controls for the second time (or more), the css class was rewrite with "[object Object]".

This bug appear with version 4.1.1. 

I provided an example in attachment 

[ExamplePradoBugWithSetStyle.zip](https://github.com/pradosoft/prado/files/6968431/ExamplePradoBugWithSetStyle.zip)

**Comment by ctrlaltca** [comment]
Hi, first thank you for the report and sorry for the delay of my reply (due to holidays).
I've tried to run your testcase but it look slime i can't replicate the problem.
When pressing the button i correctly see the color change of the outline.
The callback response contains:
```json
{"Prado.Element.j":["ctl0_Main_ctrl1","css",[{"outline":"12px solid red"}]]}
```
I've tested both prado 4.1.1 and master.

To be honest we already fixed a similar problem in master, but I'm not sure if your problem is the same.
If you want to test the fix already included in master, have a look at this commit: https://github.com/pradosoft/prado/commit/b31e08f0655a773a3d58e15462ade263ce3414c4

Thank you

**Comment by ctrlaltca** [comment]
Sorry, i forgot to ask, what php version you are running?

**Comment by zendre4** [comment]
Hi, Yes the new outline are correct but the css class are not correct after that. In My example we lost the italic class... 

I use php 7.4 with prado 4.1.1 but I'm sure at 99% percent than Php wasn't in cause, see my PR #801 

PS. in the old 4.0 version the bug not appear

**Comment by ctrlaltca** [comment]
Sorry for the delay (again).
On 4.1.1 i can reproduce the problem: after the callback the text is no more italic.
On master (future 4.2) I can't reproduce the bug, so it's probably already fixed by https://github.com/pradosoft/prado/commit/b31e08f0655a773a3d58e15462ade263ce3414c4

**Comment by ctrlaltca** [comment]
I just released prado 4.1.2 that contains the fix. Can you please try it?

**Comment by zendre4** [comment]
It's working, thanks !
